tools/windows-node-installer/test/e2e/clusterconfig/role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: RoleBinding
```

Comment:
These files are not specific to e2e suite. We are in a way suggesting how to run wni within a pod.